manifests/install.pp

```diff
@@ -81,9 +81,18 @@
     }
 
 ->
+    # ensure the GitHub SSH host authenticity is handled before we check out anything
+    exec { "Adding the fingerprint for GitHub so we can connect to it":
+        command   => "ssh -T -oStrictHostKeyChecking=no git@github.com",
+        returns   => [0,1],
```

Comment:
Why is this zero or one?  Shouldn't it just be 0?  If it returns 1, something failed, and likely the following checkout steps will also fail.  Or, am I misunderstanding?